### PR TITLE
Add test cases for cross device transition between browsers with different liveness support

### PR DESCRIPTION
**Pull request by bartzet**
# Problem
Missing test cases for hotfix: https://github.com/onfido/onfido-sdk-ui/pull/533

# Solution
Test cases have been added.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-538-pr-onfido-sdk-ui-onfido.surge.sh

File: MANUAL_REGRESSION.md [hunk 1]
```diff
@@ -242,15 +262,15 @@ Outcome:
 5. Click `Enable webcam`
 6. You should see the capture screen and camera permissions prompt
 
-##### 19. Check permission priming screen does not display when webcam is available and permission was already granted
+##### 20. Check permission priming screen does not display when webcam is available and permission was already granted
 (on Firefox, Safari and Microsoft Edge browsers)
```
Comment (stefaniacardenas) [inline MANUAL_REGRESSION.md]: There browsers display the screen multiple times, unless the user has changed the browser settings to remember the choice. Applause keeps flagging this as a bug, so I think we should change it

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-538-pr-onfido-sdk-ui-onfido.surge.sh

File: MANUAL_REGRESSION.md [hunk 2]
```diff
@@ -242,15 +262,15 @@ Outcome:
...
+(on Chrome)
 
 1. Go through the flow to document capture
 2. Upload a valid document
 3. Click `Confirm`
 4. You should see the capture screen
 
-##### 20. Check permission denied / recovery screen displays when webcam is available and permission wasn't previously denied and is denied after prompt
+##### 21. Check permission denied / recovery screen displays when webcam is available and permission wasn't previously denied and is denied after prompt
 (on Firefox, Safari and Microsoft Edge browsers)
```
Comment (stefaniacardenas) [inline MANUAL_REGRESSION.md]: Also here it should be tested on Chrome only

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-538-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://538-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**